bridge/setu/broadcaster/broadcaster.go

```diff
@@ -57,9 +58,10 @@ func NewTxBroadcaster(cdc *codec.Codec) *TxBroadcaster {
 }
 
 // BroadcastToHeimdall broadcast to heimdall
-func (tb *TxBroadcaster) BroadcastToHeimdall(msg sdk.Msg) error {
+func (tb *TxBroadcaster) BroadcastToHeimdall(msg sdk.Msg, event interface{}) error {
 	tb.heimdallMutex.Lock()
 	defer tb.heimdallMutex.Unlock()
+	defer util.LogElapsedTimeForStateSyncedEvent(event, "BroadcastToHeimdall", time.Now())
```

Comment:
Could we change LogElapsedTimeForStateSyncedEvent to receive only 1 type?

Reply:
Not sure if that's worth, as - since the method is invoked also by functions using other event types - I would need to check if the type is `statesender.StatesenderStateSynced` in many other places